src/Sylius/Bundle/AdminBundle/Resources/views/Shared/pagerfanta.html.twig

```diff
@@ -0,0 +1,59 @@
+{%- extends '@Pagerfanta/default.html.twig' -%}
```

Comment:
Extend `@BabDevPagerfanta/default.html.twig` instead.

The `@Pagerfanta` Twig namespace comes from the base `pagerfanta/twig` package and the `@BabDevPagerfanta` namespace is the Symfony bundle, the bundle is where translations are provided out-of-the-box whereas the root package uses hardcoded English defaults.